packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/translations/Messages_fr_FR.json

```diff
@@ -25,6 +25,9 @@
   "hosting_multisite_git_association_guide_name": "association de dépôt git",
   "hosting_multisite_git_association_guide_configure_webhook": "Configurer le webhoook sur Github",
   "hosting_multisite_git_association_guide_configure_tutorial": "Tutoriel de configuration",
+  "hosting_multisite_git_association_reconfigure_success": "Vous devez redéployer votre branche afin que les modifications soient prises en compte",
+  "hosting_multisite_git_association_reconfigure_warning_url_tooltip": "L'url du dépot ne peut etre modifiée. Pour renseigner une nouvelle url sur ce répertoire vous devez au préalable supprimer la configuration Git.",
+  "hosting_multisite_git_association_reconfigure_error": "La modification à échoué veuillez ré-essayer",
```

Comment:
I think we don't say "retry" if we don't know why and if it's needed to retry (if the customer isn't granted no need retry because will never works).

Reply:
Good catch, I just copied the provided message, will remove the retry part then!